packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -119,30 +119,21 @@ - (void)updateFromPlatformMarker:(FGMPlatformMarker *)platformMarker
   [self setConsumeTapEvents:platformMarker.consumeTapEvents];
   [self setPosition:FGMGetCoordinateForPigeonLatLng(platformMarker.position)];
   [self setRotation:platformMarker.rotation];
-  [self setVisible:platformMarker.visible];
   [self setZIndex:platformMarker.zIndex];
-}
-
-- (void)interpretInfoWindow:(NSDictionary *)data {
```

Comment:
This was dead code; I accidentally dropped the call to this method when converting the method above, instead of updating it (and folding it in as I've now done, since it's no longer complex enough to benefit from a separate method). That dropped call was the source of the regression.